bootstrap.sh

```diff
@@ -311,14 +308,13 @@ RUBY
 software_update() {
   logn "Checking for software updates:"
   updates=$(softwareupdate -l 2>&1)
-  # shellcheck disable=SC2086
-  if $updates | grep $Q "No new software available."; then
+  if echo "$updates" | grep "$Q" "No new software available."; then
```

Comment:
It seems that `echo` is quite necessary. This fixes a minor bug.

Reply:
will `echo "$updates"` always evaluate to true?

Reply:
`echo "$updates` will not evaluate to true. It simply prints some output and we grep it. If there's a match we reach truthfulness.